db/migrate/20181213153332_add_canonical_domain_to_searchgov_domains.rb

```diff
@@ -0,0 +1,5 @@
+class AddCanonicalDomainToSearchgovDomains < ActiveRecord::Migration
+  def change
+    add_column :searchgov_domains, :canonical_domain, :string
+  end
+end
```

Comment:
This needs to be a pull request on its own, migrated then the code pushed after it. Otherwise it may cause issues.

Reply:
will do